EnvironmentSetup/Azure/Source/kafka/kafka-persistent.yaml

```diff
@@ -31,16 +30,15 @@ spec:
       offsets.topic.replication.factor: 1
       transaction.state.log.replication.factor: 1
       transaction.state.log.min.isr: 1
-      log.message.format.version: 2.8
-      inter.broker.protocol.version: 2.8
+      log.message.format.version: "3.0"
+      inter.broker.protocol.version: "3.0"
     storage:
       type: jbod
       volumes:
       - id: 0
         type: persistent-claim
         size: 10Gi
         deleteClaim: false
-        class: was-azurefile
```

Comment:
Which storageclass are you planning to use? What if there is no default? (The deployment will fail)